front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx

```diff
@@ -64,7 +65,7 @@ function CreateShipmentView() {
   const { globalPreferences } = useContext(GlobalPreferencesContext);
 
   // variables in URL
-  const baseId = globalPreferences.selectedBase?.id!;
+  const { baseId } = useBaseIdParam();
 
```

Comment:
@fhenrich33 in l. 109 pls replace `globalPreferences.organisation?.id;` with `currentBase?.organisation?.id` because then you can remove the globalPreferences context fully

Reply:
Notice that I'm only renaming `baseId` from `useBaseIdParam()` hook. So I can't really do that. 

However, there's a task to replace the global context to address that (or I can do it right here as this is related).